dependencies.yaml

```diff
@@ -1,85 +1,86 @@
-  versionConflictPolicy: highest
-  transitivity: runtime_deps
-  resolvers:
-    - id: mavencentral
-      type: default
+  languages: [ "java", "scala:2.11.8" ]
+  resolvers: 
+    - id: "mavencentral"
+      type: "default"
       url: https://repo.maven.apache.org/maven2/
+  transitivity: runtime_deps
+  versionConflictPolicy: highest
 
 dependencies:
-  org.eclipse.aether:
-    aether:
-      version: "1.0.2.v20150114"
-      modules: [api, impl, connector-basic, transport-file, transport-http]
+  com.chuusai:
+    shapeless:
+      lang: scala
+  
+  com.fasterxml.jackson.dataformat:
+    jackson-dataformat-yaml:
       lang: java
-
+      version: "2.5.3"
+  
+  com.github.mpilquist:
+    simulacrum:
+      exports: [ "org.typelevel:macro-compat" ]
+      lang: scala
+  
   io.circe:
     circe:
-      version: "0.5.0-M2"
-      modules: [core, jackson, parser]
       lang: scala
-    circe-generic:
+      modules: [ "core", "jackson", "parser" ]
       version: "0.5.0-M2"
+    circe-generic:
+      exports: [ "com.chuusai:shapeless", "org.typelevel:cats-core", "org.typelevel:cats-kernel",
+        "org.typelevel:macro-compat" ]
       lang: scala
-      exports:
-        - "com.chuusai:shapeless"
-        - "org.typelevel:cats-core"
-        - "org.typelevel:cats-kernel"
-        - "org.typelevel:macro-compat"
-
+      version: "0.5.0-M2"
+  
   org.apache.maven:
     maven-aether-provider:
-      version: "3.1.0"
       lang: java
-
-  com.fasterxml.jackson.dataformat:
-    jackson-dataformat-yaml:
-      version: 2.5.3
+      version: "3.1.0"
+  
+  org.eclipse.aether:
+    aether:
       lang: java
-
+      modules: [ "api", "connector-basic", "impl", "transport-file", "transport-http" ]
+      version: "1.0.2.v20150114"
+  
+  org.rogach:
+    scallop:
+      lang: scala
+      version: "2.1.1"
+  
+  org.scala-lang.modules:
+    scala-xml:
+      lang: scala
+      version: "1.0.6"
+  
   org.scalacheck:
     scalacheck:
-      version: 1.12.6
       lang: scala
-
+      version: "1.12.6"
+  
   org.typelevel:
     cats:
-      version: 0.6.0
       lang: scala
-      modules: [kernel, macros, free]
+      modules: [ "free", "kernel", "macros" ]
+      version: "0.6.0"
     cats-core:
-      version: 0.6.0
+      exports: [ "org.typelevel:cats-kernel" ]
       lang: scala
-      exports: ["org.typelevel:cats-kernel"]
-
+      version: "0.6.0"
     macro-compat:
       lang: scala
 
-  com.chuusai:
-    shapeless:
-      lang: scala  # we are not explicitly using this, but need to declare it as scala
-
-  com.github.mpilquist:
-    simulacrum:
-      lang: scala # not using explicitly, declaring as scala
-      exports: ["org.typelevel:macro-compat"]
-
-  org.scala-lang.modules:
-    scala-xml:
-      lang: scala
-      version: "1.0.6"
-
 replacements:
   org.scala-lang:
     scala-library:
-      lang: scala/unmangled # scala-library is not mangled like sbt does with other jars
+      lang: "scala/unmangled"
```

Comment:
for language I could probably omit them. I can make that change. `version` is often ambiguous with a number: `version: 1.2` which breaks things. So always using a quote for a version is a good idea. I could do something like, if it does not parse as a number don't use the quote, else do, but that seems ugly?